### PR TITLE
Add information on how to whitelist ClamAV signatures (see #2982)

### PR DESCRIPTION
**Pull request by mamash**

### CODE REVIEW

File: docs/u_e-rspamd.md [hunk 1]
```diff
@@ -70,3 +70,19 @@ Simply add the line:
...
+
+This line confirms that such was identified:
+
+`clamd-mailcow_1      | Sat Sep 28 07:43:24 2019 -> instream(local): PUA.Pdf.Trojan.EmbeddedJavaScript-1(e887d2ac324ce90750768b86b63d0749:363325) FOUND`
+
+To whitelist this particular signature (and enable sending this type of file attached), add it to the ClamAV signature whitelist file:
+
+`echo 'PUA.Pdf.Trojan.EmbeddedJavaScript-1' >> data/conf/clamav/whitelist.ign2`
+
+Then restart the clamd-mailcow service.
```
Comment (MAGICCC) [inline docs/u_e-rspamd.md]: Can you add how to restart the clamd container, then I would merge it :)

Comment (mamash) [reply docs/u_e-rspamd.md]: OK, done.

**Comment by MAGICCC** [comment]
Merci

**Comment by andryyy** [comment]
Thank you very much. :)

**Pull request merged**